RevenueCatUI/Templates/Template7View.swift

```diff
@@ -462,14 +465,19 @@ struct Template7View: TemplateViewType {
 @available(iOS 13.0, tvOS 13.0, macOS 10.15, watchOS 6.2, *)
 private extension PaywallData.Configuration.Colors {
 
-    var featureIcon: Color { self.accent3Color }
-    var selectedOutline: Color { self.accent3Color }
-    var unselectedOutline: Color { self.accent2Color }
+    var featureIcon: Color { self.accent1Color }
+    var selectedOutline: Color { self.accent2Color }
+    var unselectedOutline: Color { self.accent3Color }
     var selectedDiscountText: Color { self.text2Color }
     var unselectedDiscountText: Color { self.text3Color }
     var selectedTier: Color { self.accent1Color }
     var callToAction: Color { self.selectedTier }
 
+    var tierControlBackground: Color { self.tierControlBackgroundColor ?? self.accent1Color }
+    var tierControlForeground: Color { self.tierControlForegroundColor ?? self.text1Color }
+    var tierControlSelectedBackground: Color { self.tierControlSelectedBackgroundColor ?? self.unselectedDiscountText }
+    var tierControlSelectedForeground: Color { self.tierControlSelectedForegroundColor ?? self.text1Color }
```

Comment:
These _should_ exist for a multi-tier paywall but if they don't, it uses defaults that will always be there (and match how Android also defaults)